Koha/Plugin/EDS/bootstrap/modules/eds-detail.tt

```diff
@@ -1287,7 +1287,7 @@
 [% END %]
 [% IF ( OpacStarRatings != 'disable' ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.rating.js"></script>[% END %]
 
-[% IF ( OpacHighlightedWords ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.highlight-3.js"></script>[% END %]
+[% IF ( OpacHighlightedWords ) %]<script type="text/javascript" src="[% interface %]/[% theme %]/lib/jquery/plugins/jquery.highlight-5.js"></script>[% END %]

```

Comment:
This line should be reviewed, please. Both on http://koha-2311.ebsco-gss.net/ and my test environment (VM) this file is shown as 404. Example from ebsco-gss.net

Request URL:
http://koha-2311.ebsco-gss.net/opac-tmpl/bootstrap/lib/jquery/plugins/jquery.highlight-5.js

If you look on the same site with a catalog search the file is given as:
http://koha-2311.ebsco-gss.net/opac-tmpl/lib/jquery/plugins/jquery.highlight-5_23.1105000.js

Example of inclusion of that file
https://github.com/Koha-Community/Koha/blob/23.11.x/koha-tmpl/opac-tmpl/bootstrap/en/modules/opac-results.tt

```
[% IF ( OpacHighlightedWords ) %]
        [% Asset.js("lib/jquery/plugins/jquery.highlight-5.js") | $raw %]
[% END %]
```
    
  Obviously I am not a KOHA expert, I just noticed this 404 so do with it, as you please

Reply:
confirmed